client/app/bundles/course/assessment/components/Material.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { PropTypes } from 'react';
+import { FormattedDate, FormattedTime } from 'react-intl';
+import IconButton from 'material-ui/IconButton';
+import { ListItem } from 'material-ui/List';
+import CircularProgress from 'material-ui/CircularProgress';
+import Avatar from 'material-ui/Avatar';
+import ActionAssignment from 'material-ui/svg-icons/action/assignment';
+import { standardDateFormat, shortTimeFormat } from 'lib/date_time_formats';
+
+const iconStyle = {
+  width: 32,
+  height: 32,
+  marginRight: 16,
+};
+
+const propTypes = {
+  name: PropTypes.string.isRequired,
+  url: PropTypes.string.isRequired,
```

Comment:
That time this probably need to be moved to library folder and make URL optional, or a different component, cause I also displayed dates in this Component